### PR TITLE
Added status labels to confirmation emails

### PR DESCRIPTION
**Pull request by Godmartinz**
# Description

This adds the asset's status to accepted and declined asset notifications

Accepted:
image

Declined:
image

Check-in:
image

Check out:
image

Fixes #[sc-26164] [sc-26133]

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

**Test Configuration**:
* PHP version:
* MySQL version
* Webserver version
* OS version

# Checklist:

- [ ] I have read the Contributing documentation available here: https://snipe-it.readme.io/docs/contributing-overview
- [ ] I have formatted this PR according to the project guidelines: https://snipe-it.readme.io/docs/contributing-overview#pull-request-guidelines
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by snipe** [comment]
Why you bork tests??

File: app/Notifications/CheckinAssetNotification.php [hunk 1]
```diff
@@ -162,6 +162,7 @@ public function toMail()
         $message = (new MailMessage)->markdown('notifications.markdown.checkin-asset',
             [
                 'item'          => $this->item,
+                'status'        => $this->item->assetstatus->name ?: '',
```
Comment (spencerrlongg) [inline app/Notifications/CheckinAssetNotification.php]: You should be able to use it in these spots too, @Godmartinz - just for uniformity and simplification

**Review by spencerrlongg — APPROVED** [review]
Without actually testing, this looks good to me. Let me know if you'd like me to actually test it out.

**Review by snipe — CHANGES_REQUESTED** [review]
Can we pull the indent on the new section in the mail blades? I've seen them parse funny in the past, since it's markdown and not HTML.

**Comment by Godmartinz** [comment]
indents removed 👍

**Comment by snipe** [comment]
Perfect, ty! (I never quite figured out what triggered the “sometimesy-ness” of those markdown blades failing, it would happen kinda randomly. Indents in markdown are block quotes, so I sort of get it, but was always hard to predict.)

**Pull request merged**